### PR TITLE
Cleanup form the old Chuck names

### PR DESCRIPTION
**Pull request by cortinico**
Mostly done using automated tools:

* Package name is now `com.chuckerteam.chucker...`
* All the previous occurrences of Chuck have been renamed.

### CODE REVIEW

File: library/src/main/java/com/chuckerteam/chucker/api/internal/data/room/ChuckerDatabase.kt [hunk 1]
```diff
@@ -18,7 +18,7 @@ internal abstract class ChuckerDatabase : RoomDatabase() {
         private val DB_NAME = "chucker.db"
 
         fun create(context: Context): ChuckerDatabase {
-            // We eventually delete the old DB if a previous version of Chuck/Chucker was used.
+            // We eventually delete the old DB if a previous version of Chucker/Chucker was used.
```
Comment (olivierperez) [inline library/src/main/java/com/chuckerteam/chucker/api/internal/data/room/ChuckerDatabase.kt]:
```suggestion
            // We eventually delete the old DB if a previous version of Chucker was used.
```

File: library/src/main/java/com/chuckerteam/chucker/api/internal/support/SQLiteUtils.java [hunk 2]
```diff
@@ -47,8 +47,8 @@ private static String extractDatabase(Context context) {
             File external = context.getExternalFilesDir(null);
             File data = Environment.getDataDirectory();
             if (external != null && external.canWrite()) {
-                String dataDBPath = "data/" + context.getPackageName() + "/databases/chuck.db";
-                String extractDBPath = "chuckdb.temp";
+                String dataDBPath = "data/" + context.getPackageName() + "/databases/chucker.db";
+                String extractDBPath = "chuckerdb.temp";
```
Comment (olivierperez) [inline library/src/main/java/com/chuckerteam/chucker/api/internal/support/SQLiteUtils.java]: Do we still use this method? `extractDatabase`

Comment (cortinico) [reply library/src/main/java/com/chuckerteam/chucker/api/internal/support/SQLiteUtils.java]: Not really, I'm removing that class. You'll find it in a commit in this PR.

File: library/src/main/java/com/chuckerteam/chucker/api/internal/data/room/ChuckerDatabase.kt (hunk 1 above)
Comment (cortinico) [reply library/src/main/java/com/chuckerteam/chucker/api/internal/data/room/ChuckerDatabase.kt]: Actually the original was correct.

**Pull request merged**